src/myrtlespeech/configs/deep_speech_2_en.config

```diff
@@ -98,6 +98,9 @@ train_config {
   adam {
     learning_rate: 0.001;
   }
+  exponential_lr {
+    gamma: 0.9;
+  }
```

Comment:
Are the results on `myrtleperf` available for this? If not we should avoid updating master

Reply:
No, because I only run 1 experiment with this configuration so I should run more train experiments before reporting the results. Should I delete this part from the config file for now?